Cleanup caja tabs

mdinslage opened the pull request:
The caja tabs needed the same TLC the pluma and mate-terminal tabs did.   Tabs are now aligned to frame and checked tabs are colored correctly.

clefebvre commented:
@mdinslage can you provide screenshots to illustrate the issue/fix.

does this indicate a global issue in the theme we should fix generically (as opposed to just in mate apps)?

mdinslage commented:
**@clefebvre** This is not a global issue.  There are just a few specific programs that are needing small adjustments to the tabs.  Caja is one, I installed cinnamon last night and it looks like nemo is a little off too, but I haven't gotten to it yet.  To be honest this is just very small stuff and most users aren't going to notice, but I thought it was worth fixing.  Ill post screenshots at the bottom.

@JosephMcc  I checked the margins with gtk3-widget-factory and everything looks correct. (ill post a screenshot).  Globally I have found that if you can find a program that has tabs that span all the way to the right of the frame, they are off.  I was going to submit a PR for that separately, since its an easy fix with only two lines of code If I remember, I'll have to look at my custom fork.  I don't plan on submitting too many more PR if any.  I'll just post under issues, if you guys think that would be a better.  I just want to be helpful to the project and not annoying lol.

before
![caja0](https://user-images.githubusercontent.com/28424337/40745685-c1b68e24-641d-11e8-88d8-56459b90a94e.png)

after
![caja](https://user-images.githubusercontent.com/28424337/40745707-cf9206c2-641d-11e8-9a57-4a2f07c73880.png)

JosephMcc commented:
Don't hesitate to submit PR's. I don't use Mate nor Mint-X much so don't always come across these.

mdinslage commented:
I cleaned up the nemo window area.  I aligned the tabs like I did with caja, but I also added a border to the window pane.  This doesn't really seem important now, but once the pane separator issue is fixed then it becomes clear, because the sidebar has a border and the main window doesn't it.  I also moved down the primary tool bar bottom border a tad, I did this to eliminate a double border you get when you added the window pane border.   Here are some before and after screenshots...

before
![nemo1](https://user-images.githubusercontent.com/28424337/40823256-59a4920e-6535-11e8-915d-c31d7d0bf513.png)

after
![nemo2](https://user-images.githubusercontent.com/28424337/40823260-5fa3560e-6535-11e8-9059-6938d1930fb3.png)

clefebvre commented:
@mdinslage PRs are welcome. We're going to need all the help we can get, in particular in the following areas:

- GTK 3.22 changes (Joseph worked a lot on Mint-Y on this. Mint-X also received attention but it's not as tested/ready for 3.22)
- Symbolic icons. In an effort to be future-proof, compatible with dark themes and scaling we moved more and more towards symbolic icons. A lot of work went into Mint-Y (both the theme and the icons), Mint-X is lagging behind a bit, it might miss a few icons and symbolics might not look as good (on your last screenshot for instance, the symbolics in the nemo sidebar are completely white).
- Mint-Y Icons. the X icons have been tested for a long time, the Y ones are brand new. We're inheriting Adwaita where icons are missing.. and that doesn't always looks good.
